Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:latest
+
+RUN mkdir /FreeCodeCamp
+
+WORKDIR /FreeCodeCamp
+
+RUN npm install -g npm
```

Comment:
I'll leave that to the default and remove that line, also it would be nice to have that information in the contribution guidelines